### DISCUSSION TITLE
Should be able to view Table of Contents (TOC) on mobile

### DISCUSSION THREAD

**Discussion by jtbayly** [quarto-dev/quarto-cli#10187] [Feature Requests]
Right now it is impossible to view the TOC of a page on a small screen device or narrow window. It auto-hides, with no way to show it. There ought to be a hamburger menu to display it, just like there is a hamburger menu to display the collapsed navigation sidebar on the left.

I tried to work around this by putting the page TOC in the body, but it *still* hides the TOC if you view it on a small screen, even though you are viewing the body.

**Comment by mcanouil** [comment]
- https://github.com/quarto-dev/quarto-cli/issues/5961

**Reply by jtbayly** [reply]
I apologize. I promise I did two searches looking for the issue before I posted.

**Reply by mcanouil** [reply]
No apologies needed.
Note that you first opened  your issue on the "quarto" repository which is not the same as Quarto CLI repository.
